### PR TITLE
Homework/1-2-3

### PR DESCRIPTION
**Pull request by Zenixas**

### CODE REVIEW

**Comment by Zenixas** [comment]
sorry for pushing all 3 in one, I'm not sure how I do seperate pull requests

File: Homework1/Homework1/Program.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            double.TryParse(Console.ReadLine(), out double weight);
+
+            Console.WriteLine("What is your height?");
+            double.TryParse(Console.ReadLine(), out double height);
+
+            Print_Information(name, surname, age, weight, height);
+        }
+
+
+        static void Main(string[] args)
```
Comment (Almantask) [inline Homework1/Homework1/Program.cs]: This reads weird. Code should read like book. Entry functions should be at the top and the dependencies below them. In your case it's vice versa.

File: Homework2/Homework2/Program.cs [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+                if (index != position && temp != NewArray.Length)
+                {
+                    NewArray[temp++] = array[index];
+                }
+            }
+
+            Console.WriteLine("\nBefore: ");
+            PrintArray(array);
+            Console.WriteLine("\nAfter: ");
+            PrintArray(NewArray);
```
Comment (Almantask) [inline Homework2/Homework2/Program.cs]: This console writeline is a side effect. Unless debugging, don't write lines like this

Comment (Almantask) [reply Homework2/Homework2/Program.cs]: If you do want to, write it outside of a function

File: Homework2/Homework2/Program.cs [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+            int.TryParse(Console.ReadLine(), out int position);
+            RemoveAtAny(array, position);
+
+            Console.WriteLine("\n=== SIGN UP ===");
+            SignUp();
+
+            Console.WriteLine("\n=== LOGIN ===");
+            Login();
+
+            Console.ReadKey();
```
Comment (Almantask) [inline Homework2/Homework2/Program.cs]: Split to smaller functions

**Review by Almantask — CHANGES_REQUESTED** [review]

**Comment by Zenixas** [comment]
updated hw 1 and 2 issues

**Pull request closed without merging**